protocol-resilience-test/llcupdate1/src/main/java/fixtures/llcresi/DpgClient.java

```diff
@@ -125,7 +131,7 @@ public Response<BinaryData> putRequiredOptionalWithResponse(String requiredParam
     }
 
     /**
-     * POST a JSON or a JPEG.
+     * POST a JSON or a JPEG. You can add these to a request with {@code new RequestOptions.addHeader()}
```

Comment:
Why is this here?

Please be careful on reviewing the output of the swaggers, they serve this exact purpose.